### PR TITLE
(dsl): Support `missing` aggregation

### PR DESCRIPTION
**Pull request by petarcurcin**
Resolves `missingAgg` in #118

### CODE REVIEW

File: modules/library/src/main/scala/zio/elasticsearch/ElasticAggregation.scala [hunk 1]
```diff
@@ -169,6 +169,37 @@ object ElasticAggregation {
...
+  /**
+   * Constructs a type-safe instance of [[zio.elasticsearch.aggregation.MissingAggregation]] using the specified
+   * parameters.
+   *
+   * @param name
+   * aggregation name
+   * @param field
+   * the type-safe field for which missing aggregation will be executed
+   * @tparam A
+   * expected number type
```
Comment (dbulaja98) [inline modules/library/src/main/scala/zio/elasticsearch/ElasticAggregation.scala:181]: You can remove this.

File: modules/library/src/main/scala/zio/elasticsearch/executor/response/AggregationResponse.scala
Comment (dbulaja98) [inline modules/library/src/main/scala/zio/elasticsearch/executor/response/AggregationResponse.scala:1]: Please add here checks for Missing aggregation (in `TermsAggregationBucket` object - in two places).

**Review by dbulaja98 — CHANGES_REQUESTED** [review]
Can you, please, add a website page too?

**Comment by petarcurcin** [comment]
Provide tests for missing aggregation in aggregation bucket.

**Review by drmarjanovic — APPROVED** [review]

**Review by drmarjanovic — APPROVED** [review]

**Pull request merged**